<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>img_fusion_Dialog</class>
 <widget class="QDialog" name="img_fusion_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>img_fusion_Dialog</string>
  </property>
  <widget class="QPushButton" name="loadImg_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>图片一</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imgFusion">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>启动融合</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgTextLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>471</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fusion_method">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>融合方法</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="fusionInfo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>511</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadImg_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>图片二</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="imgPath_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="imgPath_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="fusionMethod">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>352</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>融合层数</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fusionLayer">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
